ospfd/ospf_vty.c

```diff
@@ -11713,8 +11713,9 @@ static int config_write_interface_one(struct vty *vty, struct vrf *vrf)
 				vty_out(vty, " ip ospf dead-interval ");
 
 				/* fast hello ? */
-				if (OSPF_IF_PARAM_CONFIGURED(params,
-							     fast_hello))
+				if (OSPF_IF_PARAM_CONFIGURED(params, fast_hello)
+				    && params->fast_hello
+					       != OSPF_FAST_HELLO_DEFAULT)
```

Comment:
This change fixes the frr-reload problem by omitting the `minimal hello-multiplier` part of the `ip ospf dead-interval` command, but that command is still displayed in the running configuration when it shouldn't. Ideally we should fix that instead.

`sh run` should display only commands that were explicitly configured, which excludes values learned or computed dynamically. I won't NAK this PR however as it improves the situation compared to before.

Reply:
I agree with that. The patch has been updated to output `ip ospf dead-interval` only if it has value configured explicitly.

Reply:
Thanks for the update! Merging...